infrastructure/SharedInfrastructure/src/SharedInfrastructure/DatabaseStack.cs

```diff
@@ -0,0 +1,67 @@
+using Amazon.CDK;
+using Constructs;
+using Amazon.CDK.AWS.EC2;
+using Amazon.CDK.AWS.RDS;
+using Amazon.CDK.AWS.SSM;
+
+namespace SharedInfrastructure.Production;
+
+public class DatabaseStackProps : StackProps
+{
+    public Vpc Vpc { get; set; }
+}
+
+public class DatabaseStack : Stack
+{
+    private const int DatabasePort = 1433;
+
+    public DatabaseInstance Database { get; set; }
+
+    internal DatabaseStack(Construct scope, string id, DatabaseStackProps props) : base(scope, id, props)
+    {
+        var dbSG = new SecurityGroup(this, "DatabaseSecurityGroup", new SecurityGroupProps
+        {
+            Vpc = props.Vpc,
+            Description = "Allow access to the SQL Server instance from the website",
+        });
+
+        Database = new DatabaseInstance(this, $"{Constants.AppName}SqlDb", new DatabaseInstanceProps
+        {
+            Vpc = props.Vpc,
+            VpcSubnets = new SubnetSelection
+            {
+                // We do not need egress connectivity to the internet for this sample. This
```

Comment:
This comment needs to be removed, since we discovered we do need egress when running in apprunner.